Scripts/Miscellaneous/PyOrganiser/README.md

```diff
@@ -0,0 +1,13 @@
+# Organiser application
+
+    - Add plans to your day
+    - Organise your day
+    - Keep track abouts all meetings and todos
+    
+# Installing
+
+  `pip install requirements.txt`
+  

```

Comment:
Add Steps for converting UI file to python file

Reply:
No need to convert Ui file to Py